ci/scripts/check-npm-dependencies.js

```diff
@@ -13,8 +15,16 @@ const packagesPath = path.join(rootPath, 'packages');
 
 const args = process.argv.slice(2);
 
-if (args.length < 1) throw new Error('Check npm dependencies requires 1 parameter: package name');
-const [packageName] = args;
+if (args.length < 2)
+    throw new Error('Usage: node check-npm-dependencies.js <package_name> <semver>');
+
+const [packageName, semver] = args;
+const allowedSemvers = ['patch', 'minor', 'prerelease'];
```

Comment:
note:
we should unify terms, `prerelease` `beta` `canary` all of these meaning more or less the same.

Reply:
Yes, it is nice to unify, but it is complex since `yarn bump` uses `prerelease` NPM uses `beta` and we in the current approach of releases looks like we are using canary.